src/main/resources/static/assets/js/scripts.js

```diff
@@ -0,0 +1,67 @@
+<!-- Delete Item-->
+function confirm_delete_item(id) {
+    if (window.confirm('Are you sure you want to delete?')) {
+        $.ajax({
+            type : "DELETE",
+            url : "/delete/"+ id,
+            success : function() {
+                window.location.reload();
+            },
+            error : function() {
+                alert("Something didn't work well. If the error persist, please contact us!");
```

Comment:
let's use a normal error, I don't like and don't recommend the part with "please contact us"